mobile/src/stateMachine/scenario.ts

```diff
@@ -192,7 +192,7 @@ export const MasloScenario: GlobalScenario<States> = {
             { priority: 0, target: States.Consent, condition: VM.showConsent },
             { priority: 1, target: States.OnboardingEnter, condition: VM.hasActiveOnboarding },
             { priority: 2, target: States.AskNotificationsPermissions, condition: VM.askNotifyPermissions },
-            { priority: 3, target: States.HealthConsent, condition: Platform.OS == 'ios' ? VM.needsHealthPromptIOS : VM.hasHealthPermissions },
```

Comment:
How come we are no longer checking `VM.hasHealthPermissions`?

Reply:
The new methods for android and iOS are used instead